src/components/pages/home/index.tsx

```diff
@@ -122,8 +210,8 @@ function HomePage() {
           </figure>
         </section>
 
-        <section className="flex items-center flex-col-reverse md:flex-row gap-4 mt-24 mb-20">
-          <figure className="w-full md:w-[50%]">
+        <section className="flex items-center flex-col-reverse md:flex-row gap-4 mt-30 mb-20">
+          <figure className="w-[50%] md:w-[50%]">
```

Comment:
you can write one css class here since md size have the same class

Reply:
I have implemented all of the necessary changes listed, please review.